contracts/optimistic-ethereum/OVM/bridge/messaging/Abs_BaseCrossDomainMessenger.sol

```diff
@@ -27,13 +35,18 @@ abstract contract Abs_BaseCrossDomainMessenger is iAbs_BaseCrossDomainMessenger,
     mapping (bytes32 => bool) public successfulMessages;
     mapping (bytes32 => bool) public sentMessages;
     uint256 public messageNonce;
-    address override public xDomainMessageSender;
+    address internal xDomainMsgSender = DEFAULT_XDOMAIN_SENDER;
```

Comment:
I assume the name change is necessary to avoid a shadowed variable warning between this variable and the new getter function?